packages/ringcentral-widgets-library/app/components/CheckBox/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import styles from './styles.scss';
+
+function CheckBox(
+  {
+    data,
+    selected,
+    onSelect,
+    valueField,
+    textField,
+    className,
+    ...props
+  }
+) {
+  const isListObject = !!(textField && valueField);
+  return (
+    <div className={className}>
+      {
+        data.map((item, key) => {
+          const isSelected = selected === (isListObject ? item[valueField] : item);
```

Comment:
I am wondering does this support multi-select？Or it is just a simple component in the meantime.

Reply:
Will discuss this with all you guys on meeting.